lib/vanagon/platform/deb.rb

```diff
@@ -21,9 +21,9 @@ def generate_package(project)
       # Method to generate the files required to build a debian package for the project
       #
       # @param workdir [String] working directory to stage the evaluated templates in
-      # @param name [String] name of the project
+      # @param project [Vanagon::Project] project to build a debian package of
       # @param binding [Binding] binding to use in evaluating the packaging templates
-      def generate_packaging_artifacts(workdir, name, binding)
```

Comment:
@jcoconnor I was more referring to how terrible it is that we have to modify non-windows platform files to get windows to work.